pkg/yang/entry_test.go

```diff
@@ -1297,6 +1300,112 @@ func TestFullModuleProcess(t *testing.T) {
 		wantLeaves: map[string][]string{
 			"parent": {"s", "d"},
 		},
+	}, {
+		name: "parent with grouping and with extension",
+		inModules: map[string]string{
+			"parent": `
+			module parent {
+				prefix "p";
+				namespace "urn:p";
+
+				import extensions {
+					prefix "ext";
+				}
+
+				container c {
+					ext:a-define "hello";
+					uses daughter-group {
+						ext:d-define "universe";
+					}
+				}
+
+				grouping daughter-group {
+					ext:b-define "world";
+
+					leaf l {
```

Comment:
can we name these nodes in a way that makes it easier to understand what the tests should expect?  just would be a bit easier to read the tests below IMO.

Reply:
Done